Ford: Add Mach-E

coffee-cake-isaac opened the pull request:
**Checklist**
- [x] added entry to CarInfo in selfdrive/car/*/values.py and ran `selfdrive/car/docs.py` to generate new docs
- [x] test route added to [routes.py](https://github.com/commaai/openpilot/blob/master/selfdrive/car/tests/routes.py)
- [x] route with openpilot:
- [ ] route with stock system:

No new `docs.py` due to the vehicle still being in dashcam mode.

coffee-cake-isaac committed:
Update interface.py

coffee-cake-isaac committed:
Update values.py

coffee-cake-isaac committed:
Update routes.py

coffee-cake-isaac committed:
Update override.yaml

adeebshihadeh commented on selfdrive/car/ford/values.py:
```diff
@@ -79,6 +81,8 @@ class FordCarInfo(CarInfo):
   def init_make(self, CP: car.CarParams):
     if CP.carFingerprint in (CAR.BRONCO_SPORT_MK1, CAR.MAVERICK_MK1):
       self.car_parts = CarParts([Device.threex_angled_mount, CarHarness.ford_q3])
+    if CP.carFingerprint in (CAR.F_150_MK14, CAR.MUSTANG_MACH_E_MK1):
```
```suggestion
    if CP.carFingerprint in CANFD_CAR:
```

coffee-cake-isaac committed:
Update selfdrive/car/ford/values.py

Co-authored-by: Adeeb Shihadeh <adeebshihadeh@gmail.com>

coffee-cake-isaac committed:
Update interface.py

coffee-cake-isaac commented:
@sshane I did the PR break-out today instead of tomorrow since I had a bit of time. Will tag you in the F150 Lightning one as well.

adeebshihadeh reviewed (approved):
Also looks good once it's rebased. @sshane have you already verified all those FW versions?

sshane commented:
no, haven't found time yet

sshane committed:
Merge remote-tracking branch 'upstream/master' into master-ford-mache

sshane commented:
Tried to push a rebase, but I still get 403 errors.

sshane committed:
order

sshane committed:
only have dongles for these

42f08a29af3699f4
83a4e056c7072678

from https://github.com/commaai/openpilot/pull/30444

sshane commented:
Only found two dongles in https://github.com/commaai/openpilot/pull/30444, checked and both engage, and ran one dongle through test_models, all good

sshane commented:
thanks!
